app/javascript/dashboard/i18n/locale/en/conversation.json

```diff
@@ -77,6 +77,14 @@
         "TOMORROW": "Until tomorrow",
         "NEXT_WEEK": "Until next week"
       },
+      "PRIORITY": {
+        "TITLE": "Priority",
+        "NONE": "None",
+        "URGENT": "Urgent",
+        "HIGH": "High",
+        "MEDIUM": "Medium",
+        "LOW": "Low"
+      },
```

Comment:
This can be reused elsewhere, can we move this outside `CARD_CONTEXT_MENU` and have it as it's own key directly under `CONVERSATIONS`?